radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
Doesn't seem like it's going to work as-is.  I followed the recommendations in [How can I avoid linker errors with my template classes?](https://isocpp.org/wiki/faq/templates#separate-template-class-defn-from-decl) (scroll up a couple of items for explanation of the base problem with templates). 

```cpp
// layout.cpp
77: template<class T>
78: void BaseLayoutFactory<T>::drawThumb(uint16_t x, uint16_t y, uint32_t flags) const
79: {
80:   lcdDrawBitmapPattern(x, y, bitmap, flags);
81: }
82: 
83: template class BaseLayoutFactory<Layout>;
```

But still get errors down the line due to the virtual functions:

```
In file included from radio\src\gui\480x272\layout.cpp:21:0:

radio\src\gui\480x272\layout.h: In instantiation of 'Layout* BaseLayoutFactory<T>::create(WidgetsContainer<10, 10>::PersistentData*) const [with T = Layout]':

radio\src\gui\480x272\layout.cpp:83:16:   required from here

radio\src\gui\480x272\layout.h:95:51: error: cannot allocate an object of abstract type 'Layout'

radio\src\gui\480x272\layout.h:31:7: note:   because the following virtual functions are pure within 'Layout':

In file included from radio\src\gui\480x272\layout.h:24:0,

                 from radio\src\gui\480x272\layout.cpp:21:

radio\src\gui\480x272\widgets_container.h:124:26: note: 	unsigned int WidgetsContainer<N, O>::getZonesCount() const [with int N = 10; int O = 10]

radio\src\gui\480x272\widgets_container.h:126:18: note: 	Zone WidgetsContainer<N, O>::getZone(unsigned int) const [with int N = 10; int O = 10]

In file included from radio\src\gui\480x272\layout.cpp:21:0:

D:\Devel\RC-Tx\OpenTx\2.2\radio\src\gui\480x272\layout.h: In instantiation of 'Layout* BaseLayoutFactory<T>::load(WidgetsContainer<10, 10>::PersistentData*) const [with T = Layout]':

D:\Devel\RC-Tx\OpenTx\2.2\radio\src\gui\480x272\layout.cpp:83:16:   required from here

D:\Devel\RC-Tx\OpenTx\2.2\radio\src\gui\480x272\layout.h:102:51: error: cannot allocate an object of abstract type 'Layout'

D:\Devel\RC-Tx\OpenTx\2.2\radio\src\gui\480x272\layout.h:31:7: note:   since type 'Layout' has pure virtual functions
```